### PR TITLE
Fix Resource cluster API timeout

### PR DESCRIPTION
**Pull request by Andyz26**
### Context

Fix slow APIs (update/upgrade) gateway timeout.

### Checklist

- [ ] `./gradlew build` compiles code correctly
- [ ] Added new tests where applicable
- [ ] `./gradlew test` passes all tests
- [ ] Extended README or added javadocs where applicable

### CODE REVIEW

**Review by sundargates — COMMENTED** [review]
This does not fix the problem IMO. Imagine you lose the event because the node restarted after you have acked the incoming request. 

I think the right way to solve this would be increase the timeout in these cases. This way when you have acked the request, there's no case of potential loss of event.

**Comment by Andyz26** [comment]
> This does not fix the problem IMO. Imagine you lose the event because the node restarted after you have acked the incoming request.
> 
> I think the right way to solve this would be increase the timeout in these cases. This way when you have acked the request, there's no case of potential loss of event.

Synced offline.

File: mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterScalerActor.java [hunk 1]
```diff
@@ -277,6 +278,12 @@ private void onTriggerClusterRuleRefreshRequest(TriggerClusterRuleRefreshRequest
         this.fetchRuleSet();
     }
 
+    private void onQueueClusterRuleRefreshRequest(QueueClusterRuleRefreshRequest req) {
+        log.info("{}: Queue a request to refresh cluster rules", this.clusterId);
+        self().tell(new TriggerClusterRuleRefreshRequest(this.clusterId), self());
+        getSender().tell(Ack.getInstance(), self());
+    }
+
```
Comment (sundargates) [inline mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterScalerActor.java:286]: What's the need for a separate message? How is this different from sending an ack back as the first thing in TriggerClusterRuleRefreshRequest message handler?

**Review by sundargates — APPROVED** [review]
LGTM, but we can avoid the extra message because it only adds to the complexity of these actors.

File: mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterScalerActor.java (hunk 1 above)
Comment (Andyz26) [reply mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterScalerActor.java:286]: will switch to debug.

**Pull request merged**